pkgs/by-name/wr/wrangler/package.nix

```diff
@@ -10,6 +10,7 @@
   llvmPackages,
   musl,
   xorg,
+  nix-update-script,
```

Comment:
Use gitUpdater

```nix
passthru.updateScript = gitUpdater { rev-prefix = "wrangler@"; };
```

Reply:
do you know how to update `pnpmDeps.hash` with `gitUpdater`?
i searched in the repo and `nix-update-script` seems to work well. https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20pnpmDeps%20updateScript&type=code

Reply:
In my experiences, it always updates automatically.

Reply:
I've tried it locally and only `src.hash` is updated. Perhaps `nixpkgs-update` bot can handle that on its own?
I updated the PR. Anyway we can check the logs and revisit this again if it doesn't work. Thanks!